my_game.py

```diff
@@ -316,11 +318,12 @@ def setup(self):
         )
         
         # Temporary Asteroid test
-        new_asteroid = Asteroid(
-            center_x=SCREEN_WIDTH//2,
-            center_y=SCREEN_HEIGHT//2
-        )
-        self.asteroid_list.append(new_asteroid)
+        for r in range(5):
+            new_asteroid = Asteroid(
+            center_x=0,
```

Comment:
Asteroid should not accept center_x and center_y as inputs anymore.

Reply:
Jeg ved ikke hvorfor den gør det. Jeg gætter på det er, fordi vi fjernede x=0 og y=0 oppe ved Astroids